### DISCUSSION TITLE
一键同步提示服务器内部错误，请联系管理员。官网给的三个地址都不能正常拉取

### DISCUSSION THREAD

**Discussion by zhangjiechen** [fofapro/vulfocus#304] [Q&A] [answered]
**报错信息**
![图片](https://user-images.githubusercontent.com/52522445/186571108-05730c05-3128-43c5-81e2-bb566f6cc99b.png)
官网三个地址我都已经试过了，http://vulfocus.io/这个地址会重定向到https://vulfocus.cn，但是加入到后台配置文件后，提示需要解析SSL才可以访问
![图片](https://user-images.githubusercontent.com/52522445/186571243-a8b654d0-20e4-42f5-a4fd-26a2ae2877ca.png)

**操作系统**： 

**内核版本**：

**Python 环境版本**：

**Vulfocus 版本**：

**详细报错信息或堆栈报错信息**：

**Comment by qing-root** [comment]
改成vulfocus.cn我也是这样 -  -

**Comment by xxxxbxxxxx** [comment]
https://github.com/fofapro/vulfocus/issues/297

**Comment by zhangjiechen** [comment]
你应该没理解我的意思，或者目前你有没有进行过实操，明显不能用的解决方案 就直接复制粘贴嘛，你可不可以实操一下




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "fofapro/vulfocus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月30日(星期二) 上午10:12
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [fofapro/vulfocus] 一键同步提示服务器内部错误，请联系管理员。官网给的三个地址都不能正常拉取 (Issue #299)





 
Closed #299 as completed.

**Comment by snRNA** [comment]
请问有什么解决办法吧 。之前给的issue 根本不能解决问题 @0nise

**Comment by xxxxbxxxxx** [comment]
> 请问有什么解决办法吧 。之前给的issue 根本不能解决问题 @0nise

最新对外暴露的API：https://vulfocus.cn/api/imgs/info

**Comment by snRNA** [comment]
> > 请问有什么解决办法吧 。之前给的issue 根本不能解决问题 @0nise
> 
> 最新对外暴露的API：https://vulfocus.cn/api/imgs/inf

您好 提供的链接是 404   就算是https://vulfocus.cn/api/imgs/info 也无法同步  麻烦再确认一下吧

![image](https://user-images.githubusercontent.com/15699505/188053498-d6ce357a-bd17-4496-8599-7842b7812db1.png)

**Comment by xxxxbxxxxx** [comment]
#306

**Reply by xxxxbxxxxx** [reply]
临时方案

**Comment by xxxxbxxxxx** [comment, marked as answer]
Bug已经修复，使用最新 `master` 分支。
Docker 用户，可通过以下命令使用最新 Docker 版本：
```
docker pull vulfocus/vulfocus:0.3.2.11
```
